csun-metalab/nomi
# Nam 75 -- ENG: Writing to database photo input

*Description:*
This task is complete when you can perform CRUD operations with photos.

Update: Because the VM might not be up for media, we're going to follow photo uploads in a similar fashion to HELIX and FACULTY. Eventually, we're going to move to posting to the media uploads.
*****
*To Test:* 
* Add `IMAGE_UPLOAD_LOCATION=/home/vagrant/nameface/public/images` to your .env.
* Images uploaded from the browser should save locally to the specified path.
